packages/lib/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "../../tsconfig.base.json",
-  "include": ["src"]
+  "include": ["src"],
+    "compilerOptions": {
+      "baseUrl": "src",
+    }
```

Comment:
```suggestion
  "compilerOptions": {
    "baseUrl": "src"
  }
```

Reply:
Good 👁️ , I wonder why eslint didn't catch it